### PR TITLE
Add API console application

### PR DESCRIPTION
**Pull request by hihuz**
This PR contains all the changes from individual PR to build the API console.

I have added also one commit to update the build to include all of the new features, and one to setup proper client id for production config

If you want to test the feature locally: `cd console-src && yarn && yarn start`

Addresses: https://github.com/figureaps/figure-backend/issues/5664

### CODE REVIEW

File: console-src/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (p-janik) [inline console-src/.gitignore:1]: nit: I think we can also add 
```
npm-debug.log*
yarn-debug.log*
yarn-error.log*
```
saw it in one of branches

File: console-src/package.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "backstage",
```
Comment (johannespfeiffer) [inline console-src/package.json]: nit: let's use sth. like `console` here

File: console-src/package.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
+    "eslint-plugin-prettier": "^3.1.1",
+    "prettier": "^1.18.2",
+    "rimraf": "^3.0.0"
+  }
```
Comment (johannespfeiffer) [inline console-src/package.json:53]: Do we need to setup a `homepage` in package.json as we do not run the site in root but `/console`? see https://create-react-app.dev/docs/deployment/#building-for-relative-paths

**Review by johannespfeiffer — APPROVED** [review]
Looks good, great work!

Please see the minor issue with running in /console subfolder.

**Review by p-janik — APPROVED** [review]

File: console-src/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# Kontist API Console
```
Comment (p-janik) [inline console-src/README.md:1]: anything worth putting here?

File: console-src/.gitignore (hunk 1 above)
Comment (hihuz) [reply console-src/.gitignore:1]: True, good idea, I have added it here: 836aef59fa5d2c280f0f40a536fc4731fb60c81d

File: console-src/package.json (hunk 2 above)
Comment (hihuz) [reply console-src/package.json]: good idea 🙂 I have used `api-console` as `console` being a core node module, it was raising a warning: 0688738bfb6fa7e6db3a8c1f58c00f710cf9eeb1

File: console-src/package.json (hunk 3 above)
Comment (hihuz) [reply console-src/package.json:53]: this is probably a good idea yes, I have also updated the setup of react router to account for this subdirectory:
e38d72c8ceaa12174d148503064af0a13b8540a3 and 5ddd4f96250c4ea5c5a9af93e9bcc5c2471aa234

File: console-src/README.md (hunk 4 above)
Comment (hihuz) [reply console-src/README.md:1]: good idea, I have added some description here: 64cc128557a419779882ed6414966a2261731940

Not sure if enough, I wanted to keep it quite simple for now

**Review by p-janik — APPROVED** [review]
👍 👏

**Comment by hihuz** [comment]
> can we make it 3001 by default? 🤔 If i start this before backend i would have to do update docker container ports, not?

Great idea, plus the dev oauth client will be hardcoded with 3001 port, so it's way better to default to it: 44b0a8724f370c067336776eac56a5968dc1dd1e

**Review by p-janik — APPROVED** [review]

**Pull request merged**